src/i18n/locale/en.json

```diff
@@ -8,19 +8,19 @@
     },
     "error": {
       "label": {
-        "empty": "Category is Empty",
+        "empty": "Empty Category",
         "request_failure": "Could not load categories"
       },
       "no_categories_found": {
         "label": {
-          "hint": "You should make some from settings.",
-          "info": "No categories found!"
+          "hint": "You should make some from the settings.",
+          "info": "No categories found."
```

Comment:
this is a warning

Reply:
Yes, but it isn't an alarming error, merely informational.
The exclamation carries over to languages where it doesn't belong if left in.